src/services/NetworkService.ts

```diff
@@ -168,4 +169,18 @@ export class NetworkService {
                   websocketInjected: WebSocket,
               });
     }
+    /**
+     * It checks if a node has Websocket functioning properly to subscribe.
+     * @param url the url.
+     */
+    public async checkWebsocketConnection(url: string) {
+        const webSocket = new WebSocket(url);
+        let websocketConnectionStatus: boolean = false;
+        webSocket.onmessage = function (e) {
+            // @ts-ignore
```

Comment:
Can we avoid `// @ts-ignore`?

Reply:
it's needed here to skip the linter check.

![Screen Shot 2021-11-15 at 6 11 08 PM](https://user-images.githubusercontent.com/9143194/141815433-5489d6d7-bd53-4e58-91ad-267eb940c92d.png)

Reply:
maybe arrow function with the parameter type(at least `any`)?